pyodk/endpoints/submissions.py

```diff
@@ -159,12 +153,165 @@ def get_table(
         except PyODKError as err:
             log.error(err, exc_info=True)
             raise err
-        else:
-            response = self.session.get_200_or_error(
-                url=self.urls.get_table.format(
-                    project_id=pid, form_id=fid, table_name=table
-                ),
-                logger=log,
-                params=params,
+
+        response = self.session.response_or_error(
+            method="GET",
+            url=self.urls.get_table.format(project_id=pid, form_id=fid, table_name=table),
+            logger=log,
+            params=params,
+        )
+        return response.json()
+
+    def post(
+        self,
+        xml: str,
+        form_id: Optional[str] = None,
```

Comment:
I'm surprised to see `form_id` optional here and on other signatures here. Was this maybe left over from the manager-based chaining?

Reply:
More or less, yeah. It's a similar idea as project_id, where you could set a default on the Service class instance, then that'd be used as a default for any calls using that instance. The parameter validation checks that either the default or an input is set. Please let me know what you'd like to change if anything.

Reply:
When we discussed this item, @lognaturel was going to consider whether this was useful to keep in or not. So for now it is still in.

Reply:
I have no strong feelings about this so let's leave it in.